docs/data-sources/compute_flavors.md

```diff
@@ -53,3 +53,17 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - Indicates a data source ID.
 
 * `ids` - A list of flavor IDs.
+
+* `flavors` - List of ECS flavors details. The object structure of each flavor is documented below.
+
+The `flavors` block supports:
+
+* `id` - The ID of the flavor.
+
+* `cpu_core_count` - The number of vCPUs.
+
+* `memory_size` - The memory size in MB.
```

Comment:
the unit of **memory_size** parameter is GB, so it's better to  consistent with it.

Reply:
done